deploy/maven-repo.yaml

```diff
@@ -0,0 +1,67 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: jvm-build-maven-repo
+  namespace: test-jvm-namespace
+spec:
+  revisionHistoryLimit: 0
+  strategy:
+    type: Recreate
+  selector:
+    matchLabels:
+      app: jvm-build-maven-repo
+  template:
+    metadata:
+      labels:
+        app: jvm-build-maven-repo
+    spec:
+      serviceAccountName: pipeline
+      securityContext:
+        runAsUser: 0
+      containers:
+        - name: jvm-build-maven-repo
+          image: dzikoysk/reposilite:3.5.14
+          imagePullPolicy: IfNotPresent
+          ports:
+            - name: http
+              containerPort: 8080
```

Comment:
Is this file / deployment only used for development? In which case perhaps a more unique port should be used as 8080 is often used for standard deployments during development?

Reply:
Yes it does, as the same port is used for UI management and upload/download requests.

Reply:
I meant onto e.g. console-openshift-console.apps..... I think a route generation might be useful ? Something like https://github.com/redhat-appstudio/jvm-build-service/blob/main/deploy/console/base/route.yaml but 
```
kind: Route
apiVersion: route.openshift.io/v1
metadata:
  name: reposilite
  namespace: test-jvm-namespace
  labels:
    app: jvm-build-maven-repo
  annotations:
    openshift.io/host.generated: 'true'
spec:
  to:
    kind: Service
    name: jvm-build-maven-repo
    weight: 100
  port:
    targetPort: http
  tls:
    termination: edge
    insecureEdgeTerminationPolicy: Redirect
  wildcardPolicy: None
```

Reply:
Ah, gotcha. I'll give it a go.

Reply:
Have added route for UI to 'maven.repo.yaml'. Seems to work fine for accessing UI.

Reply:
Excellent!